app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= image_tag attachments['pieFullTanLogo.svg'].url, width: '150', align: "middle" %>
+
+<p>Hi<%= @greeting_name ? " #{@greeting_name}," : ","%></p>
+
+<p>Welcome to Pie for Providers! Get ready to increase your slice of the pie with simple child care subsidy management tools. To get started, please verify your email address by clicking the link below (you can also copy and paste it into your browser).</p>
+
+<p><%= link_to 'Confirm Account', "#{ActionMailer::Base.default_url_options[:protocol]}#{ActionMailer::Base.default_url_options[:host]}#{ActionMailer::Base.default_url_options[:port]}/confirm?confirmation_token=#{@resource.confirmation_token}", target: "_blank" %></p>
```

Comment:
**suggestion (non-blocking)**: Can we move `ActionMailer::Base.default_url_options` to the mailer and share it via `@options` or something?

Reply:
Actually, we can move the whole URL generation to a `private` method and share the confirmation URL via an instance variable.